addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -101,17 +101,25 @@ const UrlPickerUserValueWidget = InputUserValueWidget.extend({
         this.containerEl.after(linkButton);
         this.el.classList.add('o_we_large');
         this.inputEl.classList.add('text-start');
+    },
+
+    open() {
```

Comment:
Randomly commenting here, it's related to the snippet option but not this line
![image](https://github.com/odoo/odoo/assets/30048408/02db61f6-b8c5-4745-8448-d1edc139d9cc)

The url autocomplete in snippet option is not working anymore?

Reply:
Odoo 17: https://watch.screencastify.com/v/ohweJkaYACRfhYr3Tzph
Odoo 16.4: https://watch.screencastify.com/v/JORTqhJsx384RBZfKB40
Everything seems to be working fine in stable :shrug: